LinesWin.cs

```diff
@@ -48,6 +49,18 @@ class LinesWin : Window {
       }
    }
    Random R = new ();
+
+   void FillPolygon (string file) {
+      var lines = File.ReadAllLines (file).Select (a => a.Split (' ').Select (a => Convert.ToInt32 (a)).ToArray ());

```

Comment:
int.Parse is faster. Split on whitespace does not need any parameters. So you can simplify this to:
`File.ReadAllLines (file).Select (a => a.Split ().Select (int.Parse).ToArray ())`